test/base/playback_spec.js

```diff
@@ -33,4 +34,62 @@ describe('Playback', function() {
 
     expect(spy).to.have.been.calledOnce
   })
+
+  it('creates a default error if no error data is given', () => {
+    const errorData = this.basePlayback.createError()
+    const defaultError = {
+      description: '',
+      level: PlayerError.Levels.FATAL,
+      origin: 'playback',
+      scope: 'playback',
+      raw: {},
+      code: 'playback:unknown',
+    }
+
+    expect(errorData).to.deep.equal(defaultError)
+  })
+
+  it('creates a code error on the following format: name:code', () => {
+    this.basePlayback.name = 'test'
+    const error = { code: '42' }
+    const errorData = this.basePlayback.createError(error)
+
+    expect(errorData.code).to.deep.equal(`${this.basePlayback.name}:${error.code}`)
+  })
+
+  it('creates a code error on the following format: name:code', () => {
+    this.basePlayback.name = 'test'
+    const error = { code: '42' }
+    const errorData = this.basePlayback.createError(error)
+
+    expect(errorData.code).to.deep.equal(`${this.basePlayback.name}:${error.code}`)
+  })
+
+  it('default error level equals to FATAL', () => {
```

Comment:
Let's try to make `it` descriptions more like a single sentence. Something like `it("triggers a FATAL error by default")`.